Packs/Vectra_AI/Integrations/VectraDetect/test_data/single_outcome.json

```diff
@@ -0,0 +1,7 @@
+{
+    "builtin": false,
+    "category": "benign_true_positive",
+    "id": 4,
```

Comment:
```suggestion
    "id": "4",
```

Reply:
To fix https://app.circleci.com/pipelines/github/demisto/content/197794/workflows/df2c79ec-2b7d-490c-81af-bf5e2fcf6383/jobs/581561?invite=true#step-105-8